tests/testthat/helper-error.R

```diff
@@ -5,3 +5,6 @@ get_defunct_error_class <- function() {
     "defunctError"
   }
 }
+
+# Dummy to remind us to keep tests and verifications in sync
+verify_errors <- identity
```

Comment:
I don't understand the semantics of `verify_errors`.

Reply:
ah that's because it doesn't have any semantics :) I use it as a reminder that the contents have to be kept in sync with the corresponding section in the file-wide `verify_output()`.

That's because we need two streams of expectations, one the one hand the hard expectations that we do see the expected errors, and on the other hand the soft expectations about generated messages.

Ideally the file-wide output would be auto-generated from `verify_errors()`, but it's still unclear to what extent we can achieve this.

Reply:
The alternative is to keep many `verify_output()` files in each `test_that()` block, but this quickly gets out of hand in terms of consistency of file names, and where to find information. File-wide output files are much more manageable in my experience, and make it easy to quickly review all error messages thrown by a package.

Reply:
Did we discuss `expect_error(output = TRUE)` ?

Reply:
I don't think this would be sufficient because this wouldn't add the context necessary to understand the expression to the output file (e.g. object creation).